src/app/pages/charging-stations/list/charging-stations-list-table-data-source.ts

```diff
@@ -49,12 +49,13 @@ import { ChargingStationsConnectorsDetailComponent } from '../details-component/
 @Injectable()
 export class ChargingStationsListTableDataSource extends TableDataSource<ChargingStation> {
   private readonly isOrganizationComponentActive: boolean;
+  private readonly isPricingComponentActive: boolean;
   private editAction = new TableEditChargingStationAction().getActionDef();
   private smartChargingAction = new TableChargingStationsSmartChargingAction().getActionDef();
   private deleteAction = new TableDeleteChargingStationAction().getActionDef();
   private generateQrCodeConnectorAction = new TableChargingStationGenerateQrCodeConnectorAction().getActionDef();
   private canExport = new TableExportChargingStationsAction().getActionDef();
-  private viewPricingsAction = new TableViewPricingsAction().getActionDef();
+  private maintainPricingDefinitionsAction = new TableViewPricingDefinitionsAction().getActionDef();
```

Comment:
maintain (maintainPricingDefinitionsAction) <> view (TableViewPricingDefinitionsAction)
View is R/O, Maintain is R/W (should be Edit btw)